CupOfJoin5_2.py

```diff
@@ -0,0 +1,11 @@
+def join(*lists,sep='-'):
+    if lists==None:
+        return  None
+    newList=[]
+    for l in lists:
```

Comment:
1. You should try using a more informative name for the variable "l" if possible.
2. Is there a way to simplify the for loop? (for example, by using list comprehension?)